app/faucet/views.py

```diff
@@ -24,98 +43,63 @@ def faucet(request):
 
 
 def check_github(profile):
-    if settings.ENV == 'local':
-        creds = False
-    else:
-        creds = True
     user = search_github(profile + ' in:login type:user')
-    if len(user['items']) == 0 or user['items'][0]['login'].lower() != profile.lower():
-       response = {
-        'status': 200,
-        'user': False
-      }
-    else:
-       response = {
-        'status': 200,
-        'user': user['items'][0]
-      }
+    response = {'status': 200, 'user': False}
+    if not len(user['items']) == 0 or not user['items'][0]['login'].lower() != profile.lower():
+        response['user'] = user['items'][0]
     return response
 
 
 @csrf_exempt
 def save_faucet(request):
-    try:
-        validate_slug(request.POST.get('githubProfile'))
-    except Exception as e:
-        return JsonResponse({
-          'message': e.messages[0]
-        }, status=400)
+    github_profile = request.POST.get('githubProfile')
```

Comment:
did not test here either.. can you do a runthrough of testing?

Reply:
It's doing the same exact thing, only catching in one place versus three since we're returning on exception anyway.